components/service/glean/src/main/java/mozilla/components/service/glean/error/ErrorRecording.kt

```diff
@@ -68,7 +68,7 @@ object ErrorRecording {
 
         logger.warn("${metricData.identifier}: $message")
 
-        // There are two reasons for using `CountersStorageEngine.record` below
+        // There are two reasons for using `CountersStorageEngine.add` below
```

Comment:
Let's keep the discussion in one place, I started a review [here](https://github.com/mozilla-mobile/fenix/pull/2749#discussion_r286678555)

Reply:
I moved the pings into a-c, so that PR can be closed. Thanks for all the feedback there!